README.md

```diff
@@ -45,6 +45,54 @@ Yes! This work is still in early development, which means it's _not ready for pr
 
  [![asciicast](https://asciinema.org/a/471709.svg)](https://asciinema.org/a/471709)
 
+## Workspaces in v0.5.0
```

Comment:
don't like version references. This text should be independent and updated when something changes.